bootstrapvz/plugins/file_copy/README.rst

```diff
@@ -0,0 +1,26 @@
+File copy
+---------
+
+This plugin lets you copy files from the host to the VM under construction,
+  create directories, and set permissions and ownership.
+
+Note that this necessarily violates the `first development guideline`_.
```

Comment:
Yes, it's indeed pretty convenient.
I put that part in to make things more understandable for prospective contributors, but it can be removed.